docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: '3.7'
+
+services:
+  minio:
+    image: minio/minio:${MINIO_VERSION:-latest}
+    volumes:
+      - minio-data:/data
+      - minio-config:/root/.minio
+      - minio-events:/tmp/events
+    ports:
+      - "${MINIO_UI_PORT:-9000}:9000"
+    environment:
+      - MINIO_REGION=${MINIO_REGION:-us-east-1}
```

Comment:
@SamarthVP all these `${<var>:-<default>}` are an easy way to be able to quickly change different parameters of the `docker-compose.yml` without having to rewrite. `docker-compose` will read `.env` automatically, so we can use this feature to set "custom" global environments variables for our test case